source/sdk/lang/types.ooc

```diff
@@ -137,12 +137,18 @@ Cell: class <T> {
 		match T {
 			case Int =>
 				(this val as Int) toString()
+			case Long =>
+				(this val as Long) toString()
 			case UInt =>
 				(this val as UInt) toString()
+			case ULong =>
+				(this val as ULong) toString()
```

Comment:
Was it not possible to call `toString()` on `ULong` and `Long`, or was the result truncated or otherwise incorrect?